app/bergamot-translator-app-bytearray.cpp

```diff
@@ -20,44 +20,22 @@ int main(int argc, char **argv) {
   std::string config = options->asYamlString();
 
   // Route the config string to construct marian model through TranslationModel
-  TranslationModel model(config, marian::bergamot::getModelMemoryFromConfig(options));
+  auto model = std::make_shared<TranslationModel>(config, marian::bergamot::getModelMemoryFromConfig(options));
```

Comment:
Kenneth wanted the shared_pointer off, so reset to the left one? Is the shared_pointer required?

Reply:
I just made it consistent with the other non-bytearray file. I can remove it from both

Reply:
Reset to variable, no pointer was the call, which is why this bytearray file had no shared-pointer.

Reply:
Done